explorer/src/pages/TransactionDetailsPage.tsx

```diff
@@ -64,15 +66,17 @@ export function TransactionDetailsPage({ signature: raw }: Props) {
 function StatusCard({ signature }: Props) {
   const fetchStatus = useFetchTransactionStatus();
   const status = useTransactionStatus(signature);
-  const fetchDetails = useFetchTransactionDetails();
   const details = useTransactionDetails(signature);
   const { firstAvailableBlock, status: clusterStatus } = useCluster();
+  const [autoRefreshInProcess, setAutoRefreshInProcess] = React.useState<
+    boolean
+  >(false);
+
   const refresh = React.useCallback(
     (signature: string) => {
       fetchStatus(signature);
-      fetchDetails(signature);
     },
-    [fetchStatus, fetchDetails]
+    [fetchStatus]
```

Comment:
nit: how about we just get rid of the `refresh` callback altogether? It was a bad idea for me to add it in the first place because it obscures what's actually getting refreshed.